cp-attendance-hybrid/src/components/modal/attendance-modal/attendance-modal.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { Modal } from 'react-native';
+import { ButtonArea, ModalBackground, ModalContainer, ModalText } from './attendance-modal.component.style';
+
+import { ModalButton } from '../../button/button-modal/modal-button.component';
+import { InputField } from '../../input-field/input-field.component.style';
+import { AttendanceService } from '@src/data/service/attendance.service';
+import EmptyState from '@freakycoder/react-native-empty-state';
+import SuccessStateImage from 'assets/SuccessStateImage.png';
+import ErrorStateImage from 'assets/ErrorStateImage.png';
+
+
+interface ModalProps {
+  visible: boolean,
+  text: string,
+  close: () => void,
+  lessonId: string,
+  studentId: string,
+}
+
+export const CustomModal = (props: ModalProps) => {
+  const [content, setContent] = useState('');
+  const [error, setError] = useState(false);
+  const [success, setSuccess] = useState(false);
+  const sendPassKey = async () => {
+    const response = await AttendanceService.checkPasskey(content, props.lessonId, props.studentId);
```

Comment:
Pergunta, por que não use o .then() e o .catch()? Não achei errado só dúvida msm se tem diferença ou nem

Reply:
Pior que eu n sei se tem diferença kkkkkkkk, to fazendo assim por costume